infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -163,10 +152,26 @@ cmake_llvm $CMAKE_EXTRA_ARGS \
     -DLLVM_USE_SANITIZER=DataFlow \
     -DCMAKE_INSTALL_PREFIX=/usr/dfsan/
 
-ninja -j $NPROC cxx cxxabi
-ninja install-cxx install-cxxabi
+ninja -j $NPROC install-cxx install-cxxabi
 rm -rf $WORK/dfsan
 
+
+# AARCH64
+mkdir $WORK/aarch64-cxx
+cd $WORK/aarch64-cxx
+cmake -DLIBCXX_ENABLE_SHARED=OFF -DLIBCXX_ENABLE_STATIC_ABI_LIBRARY=ON -DLIBCXXABI_ENABLE_SHARED=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="$AARCH64_EXTRA_CFLAGS" -DCMAKE_C_COMPILER=clang -DCMAKE_EXE_LINKER_FLAGS="-fuse-ld=lld" -DCMAKE_C_COMPILER_TARGET=$AARCH64_TARGET_TRIPLE -DCMAKE_ASM_COMPILER_TARGET=$AARCH64_TARGET_TRIPLE -DCMAKE_C_COMPILER_EXTERNAL_TOOLCHAIN=$AARCH64_TOOLCHAIN -DCMAKE_SYSROOT=$AARCH64_SYSROOT -DCMAKE_INSTALL_PREFIX=$AARCH64_SYSROOT -DLLVM_ENABLE_PROJECTS="libcxx;libcxxabi" $LLVM_SRC/llvm
+make -j $NPROC install-cxx install-cxxabi
+
+mkdir $WORK/aarch64-compiler-rt
+cd $WORK/aarch64-compiler-rt
+
+cmake -G Ninja -DLIBCXX_ENABLE_SHARED=OFF -DLIBCXX_ENABLE_STATIC_ABI_LIBRARY=ON -DLIBCXXABI_ENABLE_SHARED=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="$AARCH64_EXTRA_CFLAGS" -DCMAKE_C_COMPILER=clang -DCMAKE_EXE_LINKER_FLAGS="-fuse-ld=lld" -DCMAKE_C_COMPILER_TARGET="$AARCH64_TARGET_TRIPLE" -DCMAKE_ASM_COMPILER_TARGET=$AARCH64_TARGET_TRIPLE -DCOMPILER_RT_DEFAULT_TARGET_ONLY=ON -DCMAKE_C_COMPILER_EXTERNAL_TOOLCHAIN=$AARCH64_TOOLCHAIN -DCMAKE_SYSROOT=$AARCH64_SYSROOT -DCMAKE_INSTALL_PREFIX=/usr/local/lib/clang/12.0.0/ $LLVM_SRC/compiler-rt
```

Comment:
Can we not hardcode the clang version here for DCMAKE_INSTALL_PREFIX=/usr/local/lib/clang/12.0.0/, will this break when that changes ?

Reply:
Good point. I'm not sure what's the best way to get this version number. I guess grepping output of `clang --version`?

Reply:
yes clang --version should be good to use.

Reply:
Is that path different from the default version at all? It seems to me that emitting this argument completely may have no effect? Or maybe you meant to use a non standard path, e.g. something with "aarch" prefix?